modules/common/istio/istio-values.tpl

```diff
@@ -172,11 +180,12 @@ mixer:
     autoscaleMax: 20
     resources:
       requests:
-        cpu: 64m
-        memory: 64Mi
+        cpu: 1000m
+        memory: 1G
       limits:
-        cpu: 128m
-        memory: 128Mi
+        cpu: 2000m
```

Comment:
Are we sure about this? I know that this needs a big increase but 1 CPU request and 2 CPU limit seems like overkill

Reply:
That's the recommended amount in the prometheus helm chart I believe

Reply:
all of these are the "performance tuned" requests / limits that are in the Istio helm chart by default.

In general I don't really mind high limits for some of the core istio components, but since we aren't really using a lot of the features that mixer plays a part in, I'm fine with cutting these in half